Dockerfile

```diff
@@ -16,12 +16,16 @@ WORKDIR /usr/src/multiqc
 RUN apt-get update && apt-get install -y procps && rm -rf /var/lib/apt/lists/* && \
     pip install --upgrade pip && \
     pip install -v --no-cache-dir . && \
-    find /usr/local/lib/python3.11 \( -iname '*.c' -o -iname '*.pxd' -o -iname '*.pyd' -o -iname '__pycache__' \) | xargs rm -r && \
+    apt-get clean -y && \
```

Comment:
Is there a reason to use this instead of `xargs`? Are they not doing the same thing?

Reply:
Essentially yes, but the for loop was meant to be a bit more secure when it came to things like special characters in filenames, but I've realised I needed to quote it like `for cache_dir in "$(find ... )"; do`.